script.js

```diff
@@ -0,0 +1,13 @@
+function shifumi() {
+
+    const user_choice = ['rock', 'paper', 'scissors']
+    let choice = prompt('Choose one of the following :' + user_choice[0] + ', ' + user_choice[1] + ', ' + user_choice[2])
```

Comment:
why using `let` here? you don't re-assign `choice` anywhere so `const` would be just fine

Reply:
Why not doing like L.7 with the javascript template string here? Using the backticks instead of concatenating everything using `+`?

Reply:
You are repeating a lot some stuffs here: 
`user_choice[n] + ', '`. What if we had suddenly 10 elements in the Array? This line would not work as expected anymore. 

Check Array.join :)

Reply:
thanks for the comments, I've updated accordingly :)